### ISSUE TITLE
Lighten black background brain pattern to create more visual contrast

### ISSUE THREAD

**Issue by bspeare** [uiowa/uids#299]
# Problem

The brain background black needs to be lightened slightly to create a better contrast ratio. 

<img width="417" alt="Screen Shot 2020-08-20 at 1 23 37 PM" src="https://user-images.githubusercontent.com/1036433/90810542-bf6b6d00-e2e8-11ea-8551-4b4fc7e3b795.png">

# Proposed solution

Increase opacity in the svg file.
